runtime/vm/createramclass.cpp

```diff
@@ -3115,7 +3115,7 @@ internalCreateRAMClassFromROMClassImpl(J9VMThread *vmThread, J9ClassLoader *clas
 			 *            + AccClassHasJDBCNatives (set during native method binding, not inherited)
 			 *           + AccClassGCSpecial (set during internal class load hook and inherited)
 			 *
-			 *         + AccClassIsContended (from romClass->extraModifiers and inherited)
+			 *         + AccClassContinuation (set during internal class load hook and inherited)
```

Comment:
are you renaming this flag? if so why?

Reply:
AccClassIsContended is carry-forwarded from romClass->extraModifiers, but we never use it via classDepthAndFlags( we always use ClassIsContended via romClass->extraModifiers), so we use the bit for new AccClassContinuation( the similar way for J9AccClassHasJDBCNatives vs J9AccClassIsUnmodifiable).

Reply:
does it need to be in classDepthAndFlags? can it just be a new bit in classFlags?

Reply:
we currently have J9AccClassFinalizeNeeded bit and J9AccClassOwnableSynchronizer bit in classDepthAndFlags too, for Processing/checking we can use the one mask effectively (such as in hasFinalizer()/getFlagValueForFinalizerCheck()).

Reply:
okay this needs to be done in line 3195

```
tempClassDepthAndFlags &= ~(J9AccClassHotSwappedOut | J9AccClassHasBeenOverridden | J9AccClassHasJDBCNatives | J9AccClassRAMArray | (OBJECT_HEADER_SHAPE_MASK << J9AccClassRAMShapeShift));
```